lib/tictactoe.rb

```diff
@@ -52,20 +52,35 @@ def show_board(board_spaces)
 
   def check_for_winner?(board)
     winner_symbol = ""
-    for win_combo in WIN_COMBOS do 
-      win_index_1 = win_combo[0]
-      win_index_2 = win_combo[1]
-      win_index_3 = win_combo[2]
-
-      winning_position_1 = board[win_index_1]
-      winning_position_2 = board[win_index_2] 
-      winning_position_3 = board[win_index_3] 
-
-      if (winning_position_1 == winning_position_2) && (winning_position_2 == winning_position_3)
-        if winning_position_1 != " "
-          winner_symbol = winning_position_1
-        end
+    
+    for win_combo in WIN_COMBOS do
+      row = Array.new
+      for index in win_combo do 
+        row.append(board[win_combo[index]])
```

Comment:
TypeError:
       no implicit conversion from nil to integer
       
 But I'm fairly certain that im passing in integer values @td-extreme

Reply:
oh, whoops. you probably want 
`row.append( board[index] )`

Reply:
Im an idiot. Thank you kindly Sir